File: customview/src/main/java/com/kotlin/customview/utils/UIUtils.java
package com.kotlin.customview.utils;

import android.content.Context;
import android.graphics.Typeface;

public class UIUtils {

    public static Typeface getFont(Context context, int mTextStyle) {
        switch (mTextStyle) {
            case 0:
                return FontCache.getOpenSansBoldFont(context);
            case 1:
                return FontCache.getOpenSansBoldItalicFont(context);
            case 2:
                return FontCache.getOpenSansExtraBoldFont(context);
            case 3:
                return FontCache.getOpenSansItalicFont(context);
            case 4:
                return FontCache.getOpenSansLightFont(context);
            case 5:
                return FontCache.getOpenSansRegularFont(context);
            case 6:
                return FontCache.getOpenSansSemiBoldFont(context);
            default:
                return FontCache.getOpenSansRegularFont(context);
        }
    }
}

File: customview/src/main/java/com/kotlin/customview/utils/UIUtils.kt
package com.kotlin.customview.utils

import android.content.Context
import android.graphics.Typeface

object UIUtils {

    fun getFont(context: Context, mTextStyle: Int): Typeface {
        return when (mTextStyle) {
            0 -> FontCache.getOpenSansBoldFont(context)
            1 -> FontCache.getOpenSansBoldItalicFont(context)
            2 -> FontCache.getOpenSansExtraBoldFont(context)
            3 -> FontCache.getOpenSansItalicFont(context)
            4 -> FontCache.getOpenSansLightFont(context)
            5 -> FontCache.getOpenSansRegularFont(context)
            6 -> FontCache.getOpenSansSemiBoldFont(context)
            else -> FontCache.getOpenSansRegularFont(context)
        }
    }
}
